test/plugins/test_symbols.py

```diff
@@ -27,7 +27,7 @@ def test_symbols(config):
     symbols = pyls_document_symbols(config, doc)
 
     # All four symbols (import sys, a, B, main, y)
-    assert len(symbols) == 5
+    assert len(symbols) == 4
```

Comment:
Why is the number of symbols different for Jedi 0.14? What symbol is missing now?

Reply:
The y symbol is not in the root scope, as the option `jedi_symbols: all_scopes` is False it will only return the root symbols. I will let a comment in the test. 
If we want, we can test for all symbols in all the scopes, changing the test or creating a new one.